### PR TITLE
Fix export api

### PR DESCRIPTION
**Pull request by michielderoos**
**Describe the Pull Request**

Would load forever trying to get `g.user.transfer_account`, I think this logic was created before a user could have multiple transfer accounts. Changed to make it get the primary transfer account!

**Todo**

- [x] Link relevant [trello card](https://trello.com/b/PA2LhlOh/sempo-dev) or [related issue](https://github.com/teamsempo/SempoBlockchain/issues) to the pull request
- [x] Request review from relevant @person or team
- [x] QA your pull request. This includes running the app, login, testing changes etc.
- [x] Bump [backend](https://github.com/teamsempo/SempoBlockchain/blob/e3eb0f480e86d5a8ef2c1814127b70ff018671c4/config.py#L7) and/or [frontend](https://github.com/teamsempo/SempoBlockchain/blob/e3eb0f480e86d5a8ef2c1814127b70ff018671c4/app/package.json#L3) versions following Semver

### CODE REVIEW

**Comment by tristanhcole** [comment]
[Change me_export_api to use right transfer account](https://trello.com/c/J6q6hWfr/660-change-meexportapi-to-use-right-transfer-account)

**Review by tristanhcole — APPROVED** [review]
LGTM!

**Review by enjeyw — CHANGES_REQUESTED** [review]
```
            for index, credit_transfer in enumerate(credit_transfer_list):
```
Just loading this line (not even the loop) still takes fooorrreeevvvveerrrr on my computer if i load in lots of data! I think it's what's timing out on the server.

Setting ```.enable_eagerloads(False)``` on the previous lines makes things better, but I haven't checked if there's any further optimisations by loading in transfer usages....

**Comment by michielderoos** [comment]
> ```
>             for index, credit_transfer in enumerate(credit_transfer_list):
> ```
> 
> Just loading this line (not even the loop) still takes fooorrreeevvvveerrrr on my computer if i load in lots of data! I think it's what's timing out on the server.
> 
> Setting `.enable_eagerloads(False)` on the previous lines makes things better, but I haven't checked if there's any further optimisations by loading in transfer usages....

@enjeyw I found it about 3x faster with `.enable_eagerloads(True)` rather than ` .enable_eagerloads(False)`. One thing which I think will make it way faster though, is #525  since it adds indexes on `recipient_transfer_account_id` and `sender_transfer_account_id`

File: app/server/api/export_api.py [hunk 1]
```diff
@@ -451,19 +452,19 @@ def post(self):
...
 
         if credit_transfer_list is not None:
-            file_url = export_workbook_via_s3(wb, workbook_filename, email)
+            #file_url = export_workbook_via_s3(wb, workbook_filename, email)
 
             response_object = {
                 'status': 'success',
                 'message': 'Export file created.',
-                'file_url': file_url,
+                'file_url': 'a',
```
Comment (enjeyw) [inline app/server/api/export_api.py:467]: doi

**Review by enjeyw — COMMENTED** [review]
That line is still craaazzzy slow for me if i use a large dataset... i think it's cos credit_transfers joins on transfer accounts and a bunch of other things we don't use?

**Review by enjeyw — APPROVED** [review]
LGTM!

**Pull request merged**